e2e/test/scenarios/embedding/embed-resource-downloads.cy.spec.ts

```diff
@@ -404,6 +417,95 @@ H.describeWithSnowplowEE(
             export_type: "csv",
           });
         });
+
+        it("should be able to download a static embedded question as CSV when a filter expects 1 parameter value e.g. date (metabase#58957, 59074)", () => {
+          const FILTER_VALUE = "2025-02-11";
+          const QUESTION_NAME = "Native question with a Date parameter";
+
+          // Can't figure out the type if I extracted `questionDetails` to a variable.
+          H.createNativeQuestion(
+            {
+              name: QUESTION_NAME,
+              native: {
+                "template-tags": {
+                  created_at: {
+                    id: "c9bbcc68-c59b-4ac1-b5e7-50d2123b4150",
+                    name: "created_at",
+                    "display-name": "Created At",
+                    type: "date",
+                  },
+                },
+                query:
+                  "select id, created_at, quantity from orders where created_at >= {{created_at}} limit 1",
+              },
+              parameters: [
+                {
+                  id: "c9bbcc68-c59b-4ac1-b5e7-50d2123b4150",
+                  type: "date/single",
+                  options: {
+                    "case-sensitive": false,
+                  },
+                  target: ["variable", ["template-tag", "created_at"]],
+                  name: "Created At",
+                  slug: "created_at",
+                },
+              ],
+              enable_embedding: true,
+              embedding_params: {
+                created_at: "enabled",
+              },
+            },
+            {
+              idAlias: "questionId",
+              wrapId: true,
+            },
+          );
+          cy.signOut();
+
+          cy.get("@questionId").then((questionId) => {
+            H.visitEmbeddedPage(
+              {
+                resource: { question: Number(questionId) },
+                params: {},
+              },
+              {
+                pageStyle: {
+                  downloads: true,
+                },
+              },
+            );
+          });
+
+          cy.button("Created At").should("be.visible").click();
+          DateFilter.setSingleDate(FILTER_VALUE);
+          H.popover().findByText("Add filter").click();
```

Comment:
The bug happens when we set the value via UI, we can't pass `params` in the JWT, like we did in the previous test.